src/include/btmem.h

```diff
@@ -1315,7 +1315,7 @@ struct __wt_update {
      */
     wt_timestamp_t prev_durable_ts;
 
-    WT_UPDATE *next; /* forward-linked list */
+    wt_shared WT_UPDATE *next; /* forward-linked list */
 
     uint32_t size; /* data length */
```

Comment:
size is shared as well

Reply:
`type` is written once on update creation, and `size` looks to be write once as well.  Where do you see the need for the `wt_shared` annotation?
Maybe shared is the wrong word here since we're flagging variables that require careful inter-thread syncing and shared just implies fields are visible to multiple threads.

Reply:
OK, this makes sense to me.